provisioning/miniprov/hare_mp/main.py

```diff
@@ -476,13 +524,18 @@ def cleanup_mkfs_state(utils: Utils, cns_utils: ConsulUtil):
     if not cns_utils.kv.kv_delete_in_transaction(keys):
         logging.error('Delete transaction failed for %s', keys)
 
+    logging.info("Exiting " + cleanup_mkfs_state.__name__)
+
 
 @repeat_if_fails()
 def set_mkfs_done_for(node: str, cns_utils: ConsulUtil):
+    logging.info("Entering " + set_mkfs_done_for.__name__)
     cns_utils.kv.kv_put(f'mkfs_done/{node}', 'true')
+    logging.info("Exiting " + set_mkfs_done_for.__name__)
 
 
 def init(args):
+    logging.info("Entering " + init.__name__)
     try:
         url = args.config[0]
 
```

Comment:
Let's log incase `if not is_mkfs_required(url):`
We are exiting here so better to highlight here.